README.md

```diff
@@ -110,6 +111,8 @@ export CLOUD=google && CLUSTER=demo
 otomi deploy
 ```
 
+If the values git repository does not have a remote defined yet, then otomi sets git remote origin to URL of Gitea that is deployed to the Kubernetes cluster.
```

Comment:
This one line confuses me without context. Please be aware that this README is for devs. It does not concern information about functionality.

The question is: in relation to your solution, what do you need to tell devs in order to start coding in this repo?

Reply:
I am just asking this openly, and don't have an idea yet about this...hope you do. Otherwise we can just skip adding to the readme entirely...

Reply:
Maybe something like:

### 4. Modifying otomi values.

If you are using the default values that are generated by running `otomi bootstrap` then these values are automatically synchronised with the supplied Gitea git server.

These values can be accessed through your `ENV_DIR` folder, and modifications can be committed to the supplied gitea remote.

If you have your own git hosting solution to store otomi-values in, then these will not be synchronised with Gitea.

Reply:
Point is, this README is ONLY for dev instructions, and any functionality should go to otomi.io

Reply:
Let's just leave it all out here